streams/src/test/java/org/apache/kafka/test/MockRestoreConsumer.java

```diff
@@ -85,6 +85,7 @@ public synchronized void assign(Collection<TopicPartition> partitions) {
         super.assign(partitions);
     }
 
+    @Deprecated
```

Comment:
This isn't in the `:clients` project, but it is the only other subclass of `Consumer`. I want to make sure we don't mask the deprecation.